exercises/pig-latin/src/test/java/PigLatinTranslatorTest.java

```diff
@@ -1,85 +1,157 @@
+    }
 
-                // Qu and a single preceding consonant are treated like a single consonant
-                {"square", "aresquay"},
+    @Ignore("Remove to run test")
+    @Test
+    public void testWordBeginningWithCh() {
+        // Ch is treated like a single consonant
+        assertEquals("airchay", pigLatinTranslator.translate("chair"));
+    }
 
-                // Th is treated like a single consonant
-                {"therapy", "erapythay"},
+    @Ignore("Remove to run test")
+    @Test
+    public void testWordBeginningWithQu() {
+        // Qu is treated like a single consonant
+        assertEquals("eenquay", pigLatinTranslator.translate("queen"));
+    }
 
-                // Thr is treated like a single consonant
-                {"thrush", "ushthray"},
+    @Ignore("Remove to run test")
+    @Test
+    public void testWordBeginningWithQuAndAPrecedingConsonant() {
+        // Qu and a single preceding consonant are treated like a single consonant
+        assertEquals("aresquay", pigLatinTranslator.translate("square"));
+    }
 
-                // Sch is treated like a single consonant
-                {"school", "oolschay"},
+    @Ignore("Remove to run test")
+    @Test
+    public void testWordBeginningWithTh() {
+        // Th is treated like a single consonant
+        assertEquals("erapythay", pigLatinTranslator.translate("therapy"));
+    }
 
-                // Yt is treated like a single vowel
-                {"yttria", "yttriaay"},
+    @Ignore("Remove to run test")
+    @Test
+    public void testWordBeginningWithThr() {
+        // Thr is treated like a single consonant
+        assertEquals("ushthray", pigLatinTranslator.translate("thrush"));
+    }
 
-                // Xr is treated like a single vowel
-                {"xray", "xrayay"},
+    @Ignore("Remove to run test")
+    @Test
+    public void testWordBeginningWithSch() {
+        // Sch is treated like a single consonant
+        assertEquals("oolschay", pigLatinTranslator.translate("school"));
+    }
 
-                // 'Y' is treated like a consonant at the beginning of a word
-                {"yellow", "ellowyay"},
+    @Ignore("Remove to run test")
+    @Test
+    public void testWordBeginningWithYt() {
+        // Yt is treated like a single vowel
+        assertEquals("yttriaay", pigLatinTranslator.translate("yttria"));
+    }
 
-                // 'Y' is treated like a vowel at the end of a consonant cluster
-                {"rhythm", "ythmrhay"},
+    @Ignore("Remove to run test")
+    @Test
+    public void testWordBeginningWithXr() {
+        // Xr is treated like a single vowel
+        assertEquals("xrayay", pigLatinTranslator.translate("xray"));
+    }
 
-                // 'Y' as second letter in a two letter word
-                {"my", "ymay"},
+    @Ignore("Remove to run test")
+    @Test
+    public void testYTreatedLikeAConsonantAtTheBeginningOfAWord() {
+        // 'Y' is treated like a consonant at the beginning of a word
```

Comment:
From what I can tell this comment only repeats the test title so is it really needed? 🙂 

The same applies for the comments in the next two tests 🙂